jobbergate-cli/Dockerfile

```
@@ -0,0 +1,14 @@
+FROM python:3.8-slim-buster
+
+RUN apt update && apt install -y curl libpq-dev gcc
+
+RUN curl -sSL  https://install.python-poetry.org | \
+    POETRY_HOME=/opt/poetry POETRY_VERSION=1.1.13 python && \
+    cd /usr/local/bin && \
+    ln -s /opt/poetry/bin/poetry && \
+    poetry config virtualenvs.create false
+
+COPY ./pyproject.toml ./poetry.lock* ./README* ./LICENSE* /app/
+COPY ./jobbergate_cli /app/jobbergate_cli
+WORKDIR /app
+RUN poetry install --no-dev
```

Better to put the workdir at the beginning so you get advantages of the docker layer caching

```suggestion
FROM python:3.8-slim-buster

WORKDIR /app

RUN apt update && apt install -y curl libpq-dev gcc

RUN curl -sSL  https://install.python-poetry.org | \
    POETRY_HOME=/opt/poetry POETRY_VERSION=1.1.13 python && \
    cd /usr/local/bin && \
    ln -s /opt/poetry/bin/poetry && \
    poetry config virtualenvs.create false

COPY ./pyproject.toml ./poetry.lock* ./README* ./LICENSE* /app/
COPY ./jobbergate_cli /app/jobbergate_cli

RUN poetry install --no-dev
```

---

How will setting the WORKDIR earlier in the Dockerfile help? There's nothing in workdir until the COPY command, so I don't think we would have any advantage from the layering.

---

Addressed in commit 9b55f84